aws-logs-loggroup/src/main/java/software/amazon/logs/loggroup/Configuration.java

```diff
@@ -16,6 +18,12 @@ public JSONObject resourceSchemaJSONObject() {
     }
 
     public Map<String, String> resourceDefinedTags(final ResourceModel resourceModel) {
-        return null;
+        if (CollectionUtils.isNullOrEmpty(resourceModel.getTags())) {
+            return null;
+        }
+
+        return resourceModel.getTags()
+                .stream()
+                .collect(Collectors.toMap(Tag::getKey, Tag::getValue, (value1, value2) -> value2));
```

Comment:
Nice to resolve the conflict to merge the tags into map.  The tags could have duplication and without the resolving duplication it can run into error.